src/format_utils.h

```diff
@@ -22,13 +22,85 @@
 #include <string>
 
 #include <boost/format.hpp>
+#include <boost/range/algorithm_ext/erase.hpp>
 
 #include "mococrw/error.h"
 
 namespace mococrw {
 
 namespace util {
 
+/**
+ * Helper function for reformatPEMCertificate:
+ * Searches for the next occurence of the given marker in pem starting at start
+ * @param pem String to search in
+ * @param marker Marker to search
+ * @param endPos Inidicates if start or end position of the marker shall be returned
+ * @param start Position to start searching at
+ * @throws MoCOCrWException if marker couldn't be found
+ * @return Start/End of next occurence of marker
+ */
+inline size_t findNextMarker(const std::string& pem, const std::string& marker, bool endPos, size_t start = 0)
```

Comment:
if this wasn't a temporary utility function I would complain about the boolean parameter. This is hard to read in function calls. If you read the calls in 2 weeks, you probably know why ;)

Reply:
i guess you are right, should be better now